docs/content/latest/develop/build-apps/php.md

```diff
@@ -0,0 +1,28 @@
+---
+title: PHP
+linkTitle: PHP
+description: Build a PHP App
+aliases:
```

Comment:
Don't think we need these now. This was from when the links were moved around from a different location.

Reply:
I'll defer this change because other files seem to have it also.